app/models/events.rb

```diff
@@ -0,0 +1,22 @@
+# Object representing relevant actions taken by a case  manager
+class Event
+  include Mongoid::Document
+  include Mongoid::Timestamps
+  include Mongoid::History::Trackable
+  include Mongoid::Userstamp
+  # Fields
+  field :event_type, type: String
+  field :cm_name, type: String
+  field :patient_name, type: String
+  field :patient_id, type: Integer
```

Comment:
Let's throw `line` in here too, the same way it shows up in `Patient`, if we can -- DCAF divides its work into three distinct buckets, and so we'll want to have these scopeable on line.